.github/workflows/acceptance-tests.yml

```diff
@@ -144,7 +144,8 @@ jobs:
             --set xray.masterKey=$MASTER_KEY \
             --set xray.joinKey=$JOIN_KEY \
             --set xray.serviceAccount.create=true \
-            --set xray.rbac.create=true
+            --set xray.rbac.create=true \
+            --set rabbitmq.auth.password=password@123
```

Comment:
Not good idea to expose password this way. Use GitHub secret and env var instead